jsonschema-module-microprofile-openapi-3/README.md

```diff
@@ -0,0 +1,94 @@
+18. From `@Schema(requiredProperties = …)` on fields/methods, derive its `"required"` properties.
+19. From `@Schema(minProperties = …)` on fields/methods, derive its `"minProperties"`.
+20. From `@Schema(maxProperties = …)` on fields/methods, derive its `"maxProperties"`.
+21. From `@Schema(nullable = true)` on fields/methods, include `null` in its `"type"` - when schema type is ARRAY, applied on array item only.
+22. From `@Schema(allowableValues = …)` on fields/methods, derive its `"const"`/`"enum"`.
+23. From `@Schema(defaultValue = …)` on fields/methods, derive its `"default"` - when schema type is ARRAY, applied on array item only.
+24. From `@Schema(readOnly = …)` on fields/methods, to mark them as `"readOnly"`.
+25. From `@Schema(writeOnly = …)` on fields/methods, to mark them as `"writeOnly"`.
+26. From `@Schema(minLength = …)` on fields/methods, derive its `"minLength"`.
+27. From `@Schema(maxLength = …)` on fields/methods, derive its `"maxLength"`.
+28. From `@Schema(format = …)` on fields/methods, derive its `"format"`.
+29. From `@Schema(pattern = …)` on fields/methods, derive its `"pattern"`.
+30. From `@Schema(multipleOf = …)` on fields/methods, derive its `"multipleOf"`.
+31. From `@Schema(minimum = …, exclusiveMinimum = …)` on fields/methods, derive its `"minimum"`/`"exclusiveMinimum"`.
+32. From `@Schema(maximum = …, exclusiveMaximum = …)` on fields/methods, derive its `"maximum"`/`"exclusiveMaximum"`.
+33. From `@Schema(minItems = …)` on fields/methods when schema type is ARRAY, derive its `"minItems"`.
+34. From `@Schema(maxItems = …)` on fields/methods when schema type is ARRAY, derive its `"maxItems"`.
+35. From `@Schema(uniqueItems = …)` on fields/methods when schema type is ARRAY, derive its `"uniqueItems"`.
+
+Schema attributes derived from `@Schema` on fields are also applied to their respective getter methods.
+Schema attributes derived from `@Schema` on getter methods are also applied to their associated fields.
+
+----
+
+## Documentation
+JavaDoc is being used throughout the codebase, offering contextual information in your respective IDE or being available online through services like [javadoc.io](https://www.javadoc.io/doc/com.github.victools/jsonschema-module-microprofile-openapi-3).
+
+Additional documentation can be found in the [Project Wiki](https://github.com/victools/jsonschema-generator/wiki).
+
+----
+
+## Usage
+### Dependency (Maven)
+```xml
+<dependency>
+    <groupId>com.github.victools</groupId>
+    <artifactId>jsonschema-module-microprofile-openapi-3</artifactId>
+    <version>[4.37.0,)</version>
+</dependency>
+```
+
+The release versions of the main generator library and this module are aligned.
+It is recommended to use identical versions for both dependencies to ensure compatibility.
+
+### Code
+#### Passing into SchemaGeneratorConfigBuilder.with(Module)
+
+```java
+
+
+```
```

Comment:
**suggestion**: The import references are missing, to avoid ambiguity with the sample code.

Reply:
My bad, I removed them without noticing it.
Thanks